arch/arm/boot/dts/overlays/mpu6050-overlay.dts

```diff
@@ -13,12 +13,16 @@
                         status = "okay";
                         clock-frequency = <400000>;
 
-                        inv-mpu6050@68 {
+                        mpu6050: inv-mpu6050@68 {
                                 compatible = "invensense,mpu6050";
                                 reg = <0x68>;
                                 interrupt-parent = <&gpio>;
                                 interrupts = <4 1>;
                         };
                 };
         };
+
+        __overrides__ {
+                mpu6050_interrupt_pin = <&mpu6050>,"interrupts:0";
```

Comment:
You don't need to qualify the parameter name with the device name - the parameters exposed by an overlay are only in scope until the next overlay is loaded, so it is safe to have multiple overlays all using the same parameter names. Other overlays have used `interrupt` or `int_pin`.